substratevm/src/com.oracle.svm.test/src/com/oracle/svm/test/AbstractClassSerializationTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020, 2020, Alibaba Group Holding Limited. All rights reserved.
```

Comment:
This test war written by us alone, in 2021. Thus you should use
```
Copyright (c) 2021, 2021, Oracle and/or its affiliates. All rights reserved.
```
and on other copyright holder.